client.go

```diff
@@ -244,10 +246,23 @@ func (fc *Client) CreateMediaDownloadRequest(
 	if err != nil {
 		return nil, err
 	}
+
+	return fc.doHTTPRequest(ctx, req)
+}
+
+func (fc *Client) doHTTPRequest(ctx context.Context, req *http.Request) (*http.Response, error) {
+	reqID := util.RandomString(12)
+	logger := util.GetLogger(ctx).WithField("out.req.ID", reqID)
```

Comment:
Would it make sense to set a ```WithField("server", r.fields.Destination)``` here?

Reply:
yeah I guess so. It's fairly clear from the URI, but I guess it's useful to be able to grep for the server field.